### ISSUE TITLE
Boost factor needs exaplanation

### ISSUE THREAD

**Issue by alhadhrami** [zerocracy/zerocracy.github.io#38]
In [§5](http://www.zerocracy.com/policy.html#5):

1. Why exactly is `2x` equal to 30 minutes? Doesn't it depend on the job's fixed budget?

2. What does a boost factor of `1x` mean exactly?

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this issue

**Comment by 0crat** [comment]
@alhadhrami[/z](https://www.0crat.com/u/alhadhrami) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/C3NDPUA8L) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by g4s8** [comment]
@alhadhrami the minimum job estimate is 15 min here, but default issue estimate is 30 min, we're using `1x` boost for PR jobs, because reviewer gets payment for 15 min work.

Also ARC or PO can change boost manually, e.g. `boost 1x` will set 15 min estimate for a job, `boost 4x` - 60 min. So you can read boost factor as `{FACTOR}x = {FACTOR} * 15min`.

**Comment by alhadhrami** [comment]
@g4s8 thanks for the explanation.
Given that the policy already mentions a fixed budget in [§4](http://www.zerocracy.com/policy.html#4), I think a rewording of this is needed in order to provide a better explanation. I suggest to look at:
- The name "boost" indicates, at least to me, a multiplication on the original job estimation. Example: a boost factor of `2x` on a 30 min job means that the budget of this task is now doubled, as in multiplied by 2, to become 60 min. Which means that a boost factor of `1x` is meaningless because the new job budget after the boost is the same, meaning that effectively there was no boost. If this is not the intended meaning, a rewording is needed.
- If the above point is rejected and no rewording happens, then at least the policy should be elaborated by the explanation given in your comment.

What do you think?

**Comment by g4s8** [comment]
@alhadhrami I agree, I also thought that `2x` boost should double task budget, but get used to now. Maybe we can rename `boost` into something else to avoid this confusion, @yegor256 what do you say?
